webapp/public/index.html

```diff
@@ -24,7 +25,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>ft_transcendence</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500;700&display=swap" rel="stylesheet">
```

Comment:
we're linking against some external font we retrieve from google api?
Would it make sense to download those fonts and keep them in assets, not to rely in availability/licensing/not changeanbility of external services?

Reply:
Do you trust yourself more than Google? 😄

Reply:
Apart from jokes, is very usual to store this kind of things in CDNs near to users (as well as images) and not directly bundled with the code, and is very common to import fonts from outside sources. I'd say importing this is safe because is an open font, and Google services are not very likely to be down

Reply:
Both are good choices, and both have pros and cons. We could change it in the future if there are any issues :+1: